plugins/eslint-plugin-liferay-portal/lib/rules/no-global-fetch.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * © 2017 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+const DESCRIPTION =
+	'Direct usage of the `fetch` API is discouraged in favour of our wrapped version that takes care of the necessary security configuration; import fetch from frontend-js-web instead';
+
+module.exports = {
+	meta: {
+		docs: {
+			description: DESCRIPTION,
+			category: 'Best Practices',
+			recommended: false,
+			url: 'https://issues.liferay.com/browse/LPS-98283',
+		},
+		fixable: null,
+		messages: {
+			noGlobalFetch: DESCRIPTION,
+		},
+		schema: [],
+		type: 'problem',
+	},
+
+	create(context) {
+		let foundFetchImport = false;
```

Comment:
I think this is sufficiently scoped per-context... if it's not, we can store this using some additional `context` information.